app/src/app/playerfilters/page.tsx

```diff
@@ -0,0 +1,99 @@
+    const [points, setPoints] = useState(-1)
+    const [games, setGames] = useState(-1)
+    const [season, setSeason] = useState(-1)
+
+    useEffect(() => {
+        fetch(`${API}/playerfilter?rebounds=${rebounds}&assists=${assists}&points=${points}&games=${games}&season=${season}`)
+          .then(response => response.json())
+          .then(data => setResults(data.data ?? []))
+          .catch(error => setResults([]))
+    }, [rebounds, assists, points, season]);
+
+    const handleKeyDown = (event: any) => {
+        console.log(results)
+        const name = event.target.name
+        const value = event.target.value.length == 0 ? -1 : event.target.value
+        if (event.key === 'Enter') {
+            switch (name) {
+                case "rebounds":
+                    setRebounds(value)
+                    break
+                case "assists": 
+                    setAssists(value)
+                    break
+                case "points": 
+                    setPoints(value)
+                    break
+                case "games": 
+                    setGames(value)
+                    break
+                case "season": 
+                    setSeason(value)
+                    break
+            }
+        }
+	}
+
+    const generateCols = () => {
+        if (results.length === 0) return (<tr><td colSpan={7} align='center'>No data found.</td></tr>)
+        return (
+            <>
+            {results.map((result, index) => (
+                <tr key={index}>
+                    <td>{result.id}</td>
+                    <td>{result.name}</td>
+                    <td>{result.rebounds}</td>
+                    <td>{result.assists}</td>
+                    <td>{result.points}</td>
+                    <td>{result.games}</td>
+                    <td>{result.season}</td>
+                </tr>
+            ))}
+            </>
+        )
+    }
+
+    return (
+        <div className={styles.settingsOuterContainer}>
+            <div className={styles.settingsContainer}>
+                <div className="d-flex justify-content-center">
+                    <input type="number" name="rebounds" className="form-control m-3" style={{ width: "120px"}} onKeyDown={handleKeyDown} placeholder="Rebounds"></input>
+                    <input type="number" name="assists" className="form-control m-3" style={{ width: "120px"}} onKeyDown={handleKeyDown} placeholder="Assists"></input>
+                    <input type="number" name="points" className="form-control m-3" style={{ width: "120px"}} onKeyDown={handleKeyDown} placeholder="Points"></input>
+                    <input type="number" name="games" className="form-control m-3" style={{ width: "120px"}} onKeyDown={handleKeyDown} placeholder="Games"></input>
+                    <input type="number" name="season" className="form-control m-3" style={{ width: "120px"}} onKeyDown={handleKeyDown} placeholder="Season"></input>
+                </div>
```

Comment:
add some text to instruct the user how to use. i.e. press enter after inputting number, filters will return results greater than values input.